src/otp-verification/contact-input-section-styles.tsx

```diff
@@ -0,0 +1,165 @@
+import styled, { css } from "styled-components";
+import { Border, Colour, Font, Motion, Spacing } from "../theme";
+import { ThemeButton } from "../theme/components/theme-helper";
+import { InputGroup } from "../input-group";
+import { Button } from "../button";
+import { PhoneNumberInput } from "../phone-number-input";
+import { InputWrapperStyleProps } from "../shared/input-wrapper";
+import { InternalOtpState } from "./types";
+
+export const ContactSectionWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+export const ContactLabel = styled.label`
+    color: ${Colour["text-subtle"]};
+    display: inline-block;
+    ${Font["form-label"]}
+    font-weight: ${Font.Spec["weight-semibold"]};
+    margin-bottom: ${Spacing["spacing-8"]};
+`;
+
+export const ContactInputSectionWrapper = styled.div`
+    display: flex;
+`;
+
+export const ContactInputWrapper = styled.div<{
+    isMaxWidth?: boolean;
+}>`
```

Comment:
add `$` for styled component props to mark them as transient

and declare the typings under standalone interfaces, refer to other files for the repo conventions